cmd/openshift-install/create.go

```diff
@@ -399,19 +403,20 @@ func waitForConsole(ctx context.Context, config *rest.Config, directory string)
 			}
 			logrus.Debug("OpenShift console route is created")
 			cancel()
-		} else if err != nil {
```

Comment:
can we switch wait for console route to UntilWithSync and drop these silenceRemaning bit...?

Reply:
> can we switch wait for console route to UntilWithSync...

Done (in a separate commit) with d3214acda -> 2f625248e.